packages/fluentui/docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -489,30 +489,32 @@ const Sidebar: React.FC<RouteComponentProps & SidebarProps> = props => {
           Fluent <span style={gradientTextStyles}>UI</span>
         </Text>
         <VersionDropdown width={props.width} />
-        <CopyToClipboard value={`yarn add ${pkg.name}@${pkg.version}`} timeout={3000}>
-          {(active, onClick) => (
-            <Box
-              as="code"
-              onClick={onClick}
-              styles={{
-                display: 'block',
-                fontWeight: 'normal',
-                fontSize: '12px',
-                opacity: active ? 1 : 0.6,
-                color: active ? 'rgb(138, 255, 124)' : 'inherit',
-                marginTop: '10px',
-                cursor: 'pointer',
-                ...(!active && {
-                  ':hover': {
-                    opacity: 0.75,
-                  },
-                }),
-              }}
-            >
-              {active ? 'Copied! Happy coding :)' : `${pkg.name}@${pkg.version}`}
-            </Box>
-          )}
-        </CopyToClipboard>
+        {process.env.NIGHTLYRELEASE ? null : (
```

Comment:
```suggestion
        {process.env.NIGHTLYRELEASE && (
```

Reply:
Proposed change will produce a different result

Reply:
Oh whoops I got the condition mixed up ! yeah in this case the ternary might be a better choice